src/backend/parser/cypher_clause.c

```diff
@@ -2786,7 +2811,7 @@ static void setNamespaceLateralState(List *namespace, bool lateral_only,
  * Code borrowed and inspired by PG's transformFromClauseItem. Static function
  * to add in the VLE function as a FROM clause entry.
  */
-static RangeTblEntry *append_VLE_Func_to_FromClause(cypher_parsestate *cpstate,
+static ParseNamespaceItem *append_VLE_Func_to_FromClause(cypher_parsestate *cpstate,
```

Comment:
[formatting] exceeds 80 columns

Reply:
Unfortunately, not everything can fit. We need to work on getting the autoformatting set up.

Reply:
oh okay 👍🏻